scripts/qa_crowbarsetup.sh

```diff
@@ -3019,6 +3085,47 @@ function oncontroller_run_tempest()
     return $tempestret
 }
 
+function oncontroller_manila_generic_driver_setup()
+{
+    local service_image_url=http://clouddata.cloud.suse.de/images/other/manila-service-image.qcow2
+    local service_image_name=manila-service-image.qcow2
+    local sec_group="manila-service"
+    local neutron_net=$sec_group
+
+    wget --progress=dot:mega -nc -O $service_image_name \
+        "$service_image_url" || complain 73 "manila image not found"
+
+    . .openrc
+    openstack image create --file $service_image_name --public \
```

Comment:
if you switch the tenant here to the one you create below, then you don't need to create the manila image as --public but it could be project specific